src/box/lua/config/cluster_config.lua

```diff
@@ -202,7 +202,13 @@ local nested_cluster_config = schema.new(schema_name, record_from_fields({
                     instance_config_with_scope('replicaset'),
                     instances = schema.map({
                         key = name,
-                        value = instance_config_with_scope('instance'),
+                        value = record_from_fields({
+                            isolated = schema.scalar({
+                                type = 'boolean',
+                                default = false,
+                            }),
+                            instance_config_with_scope('instance'),
+                        }),
```

Comment:
```
LuajitError: [instance_config] isolated: No such field in the schema
fatal error, exiting the event loop
```

Makes sense. I can't have an option related to the instance that is not in the instance config. I'll rework it.

Reply:
Solved now.